datastorages/cmis/src/main/java/org/fao/geonet/api/records/attachments/CMISStore.java

```diff
@@ -439,12 +435,8 @@ public String delResource(final ServiceContext context, final String metadataUui
                               final String resourceId, Boolean approved) throws Exception {
         int metadataId = canEdit(context, metadataUuid, approved);
         if (tryDelResource(context, metadataUuid, metadataId, visibility, resourceId)) {
-            Log.info(Geonet.RESOURCES,
```

Comment:
There were some logs that were duplicated, and they were both calling tryDelResource.
So I move the common log message into tryDelResource function so that there were no duplications.

For the filesystem store, there was not common function being called so they were added.